### PR TITLE
run through clang-tidy

### PR DESCRIPTION
**Pull request by neheb**
I think https://github.com/gerbera/gerbera/commit/d7c84cc9e967528ae91e62f513c8385248721746 introduced a bunch of clang-tidy fixes.

### CODE REVIEW

**Comment by neheb** [comment]
Oh boy. Wonder what went wrong.

**Comment by neheb** [comment]
hmm guess rebasing issue.

File: src/config/directory_tweak.cc [hunk 1]
```diff
@@ -100,7 +100,7 @@ std::shared_ptr<DirectoryTweak> DirectoryConfigList::get(const fs::path& locatio
 {
     AutoLock lock(mutex);
     for(auto testLoc = location.has_filename() ? location.parent_path() : location; testLoc.has_parent_path() && testLoc != "/"; testLoc = testLoc.parent_path()) {
-        auto entry = std::find_if(list.begin(), list.end(), [=] (const auto& d) { return ((d->getLocation() == location || d->getInherit() == true) && d->getLocation() == testLoc); }); 
+        auto entry = std::find_if(list.begin(), list.end(), [=](const auto& d) { return ((d->getLocation() == location || bool(d->getInherit())) && d->getLocation() == testLoc); });
```
Comment (KarlStraussberger) [inline src/config/directory_tweak.cc:103]: no need for a cast getInherit() returns a bool

**Review by KarlStraussberger — CHANGES_REQUESTED** [review]

File: src/config/directory_tweak.cc (hunk 1 above)
Comment (neheb) [reply src/config/directory_tweak.cc:103]: Oddly enough this change was automated...

**Pull request merged**